static/js/lib/jquery-ui/autocomplete.js

```diff
@@ -538,7 +538,13 @@ $.widget( "ui.autocomplete", {
 	},
 
 	_renderItemData: function( ul, item ) {
-		return this._renderItem( ul, item ).data( "ui-autocomplete-item", item );
+		var rendered = this._renderItem( ul, item );
+
+		// We are overwriting `_renderItem` in some places and return
+		// nothing in case of duplicate filtering.
+		if (rendered) {
+			rendered.data("ui-autocomplete-item", item )
+		}
```

Comment:
Modifying the jquery-ui files? Is there a way to avoid that? Because next time we'll update the jquery-ui lib, we'll forget about this modification, and overwrite it, right?